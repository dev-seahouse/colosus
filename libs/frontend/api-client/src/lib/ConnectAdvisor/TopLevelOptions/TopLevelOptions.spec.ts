import ConnectAdvisorTopLevelOptionsApi from './TopLevelOptions';

describe('TopLevelOptions', () => {
  const connectAdvisorTopLevelOptionsApi =
    new ConnectAdvisorTopLevelOptionsApi();

  describe('getTopLevelOptions()', () => {
    it('should return a valid response', async () => {
      const res = await connectAdvisorTopLevelOptionsApi.getTopLevelOptions();

      expect(res.data).toMatchSnapshot();
    });
  });

  describe('updateTopLevelOptions()', () => {
    it('should return a valid response', async () => {
      const res = await connectAdvisorTopLevelOptionsApi.updateTopLevelOptions({
        incomeThreshold: 10000,
        retireeSavingsThreshold: 10000,
      });

      expect(res.status).toEqual(200);
    });
  });
});
