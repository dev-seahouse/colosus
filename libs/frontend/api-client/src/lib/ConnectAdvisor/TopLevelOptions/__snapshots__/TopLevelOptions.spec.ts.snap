// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`TopLevelOptions > getTopLevelOptions() > should return a valid response 1`] = `
{
  "contactLink": "https://www.bambu.life",
  "incomeThreshold": 100000,
  "retireeSavingsThreshold": 100000,
}
`;
